ui/dashboards/src/components/DashboardToolbar.tsx

```diff
@@ -80,14 +81,18 @@ export const DashboardToolbar = (props: DashboardToolbarProps) => {
             <Typography variant="h2">{dashboardName}</Typography>
             <Stack direction="row" spacing={2} sx={{ marginLeft: 'auto' }}>
               <TimeRangeControls />
-              <Button
-                variant="outlined"
-                startIcon={<PencilIcon />}
-                onClick={onEditButtonClick}
-                sx={{ marginLeft: 'auto' }}
-              >
-                Edit
-              </Button>
+              {isLaptopSize ? (
```

Comment:
probably here there is a better way to write it. I found a bit ugly like that, specially the `else` statement that returns an empty bracket.
I'm pretty sure there is way to avoid it, but I don't remember how 😅

Reply:
maybe instead of completely removing the edit button, we should disable it and have a tooltip that tells users that we do not support edit mode for mobiles

Reply:
ah yeah perfect. Thank you @cndonovan !